restapi/src/main/java/io/stargate/web/docsapi/resources/CollectionsResource.java

```diff
@@ -0,0 +1,232 @@
+package io.stargate.web.docsapi.resources;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.stargate.db.datastore.DataStore;
+import io.stargate.db.schema.Table;
+import io.stargate.web.docsapi.dao.DocumentDB;
+import io.stargate.web.docsapi.models.DocCollection;
+import io.stargate.web.docsapi.service.CollectionService;
+import io.stargate.web.models.Error;
+import io.stargate.web.models.ResponseWrapper;
+import io.stargate.web.resources.Converters;
+import io.stargate.web.resources.Db;
+import io.stargate.web.resources.RequestHandler;
+import io.swagger.annotations.*;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Api(
+    produces = MediaType.APPLICATION_JSON,
+    consumes = MediaType.APPLICATION_JSON,
+    tags = {"documents"})
+@Path("/v2/namespaces/{namespace-id: [a-zA-Z_0-9]+}")
```

Comment:
```suggestion
@Path("/v2/namespaces/{namespace-id: [a-zA-Z_0-9]+}/collections")
```

Can't we consolidate up here and remove the `@Path("collections")` on several of the methods below?

Reply:
probably - if there is no path, do I annotate with `@Path("/")`? or `@Path("")`?

Reply:
You just leave the annotation off completely.

Reply:
It turns out I couldn't just do this...something about how the routes are prioritized in the tree with DropWizard gets messed up. If I make the above change, then all the routes in DocumentResourceV2 that have the same prefix start to 404